src/Exception/AssetIntegrationFailedException.php

```diff
@@ -0,0 +1,7 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\Monitoring\Exception;
+
+class AssetIntegrationFailedException extends \LogicException
```

Comment:
Must extend `\RuntimeException` then.

Reply:
Are you sure? The FrameworkBundle is also throwing `\LogicException`s.

![image](https://user-images.githubusercontent.com/439899/50159286-a53ce880-02d6-11e9-9217-7775803b7043.png)

Reply:
Yeah, they are throwing it when configuring the bundle. We won't do that anymore but throw it, because a runtime dependency is missing.

That is sort-of the showcase for `\RuntimeException`.